src/refiners/foundationals/ViT_matte/vit_matte.py

```diff
@@ -0,0 +1,11 @@
+import refiners.fluxion.layers as fl
+from refiners.foundationals.ViT_matte.decoder import Detail_Capture
+from refiners.foundationals.ViT_matte.vit_backbone import MViTH
+
+
+class ViTMatte(fl.Chain):
```

Comment:
Could this be merged with the "MViT" class? Or both of them are needed? If it's the case, the name should reflect their specific function.